presto-test-jdbc-compatibility/src/test/java/io/prestosql/TestJdbcResultSetCompatibility.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql;
+
+import com.google.common.collect.ImmutableMap;
+import io.airlift.log.Logging;
+import io.prestosql.jdbc.TestJdbcResultSet;
+import io.prestosql.server.testing.TestingPrestoServer;
+import org.testng.SkipException;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import java.util.Optional;
+
+import static io.prestosql.JdbcDriverCapabilities.hasBrokenParametricTimestampWithTimeZoneSupport;
+import static io.prestosql.JdbcDriverCapabilities.testedVersion;
+import static io.prestosql.TestingServerUtils.setTestingServer;
+import static org.assertj.core.api.Assertions.assertThat;
+
+@Test(singleThreaded = true)
```

Comment:
is this inheritable ?

(ie it's not declsared as such, but maybe testng traverses the hierarchy)